src/workspacer/Keybinds/KeybindManager.cs

```diff
@@ -56,9 +57,28 @@ public KeybindManager(IConfigContext context)
             thread.Start();
         }
 
+        public void ShowKeybindWarning(string warningMessage)
+        {
+            _keybindWarning = new TextBlockMessage("workspacer keybinds", "Warning, duplicate keybinds!", warningMessage, new List<Tuple<string, Action>>()
```

Comment:
We still have the issue with the supplied `warningMessage` only being used for the first invocation here, and then it gets ignored in other subsequent invocations.

Either we should use a fixed message or update `warningMessage` on each invocation.

Reply:
Sorry my programming knowledge is very limited so I don't really understand what you mean by "subsequent invocations" and why the `warningMessage` would get ignored after it was run once. When I bind multiple keys to the same shortcut the `TextBlockMessage` opens up multiple times  with the correct message.

Again sorry for not getting it

Reply:
Consider the following C# class:

````c#
public class Buggy
{
    public string LastMessage { get; set; }

    public void SetMessage(string message)
    {
        if (LastMessage == null)
        {
             LastMessage = message;
        }
    }
}
````

Now lets use this class:

````c#
var buggy = new Buggy();
buggy.SetMessage("boo");
buggy.SetMessage("hiss!");

Console.WriteLine(buggy.LastMessage);
// boo
````

Given the name of the method, this is obviously not the expected behaviour of this class, right?

If you look into your code, you have the exact same kind of bug in your implementation of `ShowKeybindWarning()`.

The fix for this problem is either:

- hard-code the message inside the function and make the function don't have any parameters at all, or
- ensure the provided parameter gets used every time, unlike now, where we only use it on the first call.

Do you understand what I mean now?